### PR TITLE
Add global rate limit flags for assets

### LINKED ISSUE #2414: Backend and agent --assets-rate-limit and --assets-burst-limit config options
### Description
New options have been added to the backend and agent to configure global rate limits for fetching assets.

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [x] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
Help usage examples for sensu-agent and sensu-backend. Config options definitions for sensu-agent and sensu-backend. Perhaps a note in the assets guide could be helpful (ex. `Note: configure rate limits for asset downloads here`).

### Expected behavior
- `assets-rate-limit` is the maximum number of assets per second that will be fetched
  - float, default 1.39 (equal to the Github API rate limits which is ~5000 user to server requests per hour)
- `assets-burst-limit` is the maximum amount of burst allowed in a rate interval
  - int, default 100 

### Current behavior
No docs yet since these are new flags.

### Corrections needed
The following flags need to be added to the backend and agent config docs and help usage.
```
      --assets-burst-limit int          asset fetch burst limit (default 100)
      --assets-rate-limit float         maximum number of assets fetched per second
```

### Additional context
https://github.com/sensu/sensu-go/pull/3716 is targeting the `release/5.19` branch but it is unknown at this time which release it will land in.

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
- Adds `--assets-burst-limit` and `--assets-rate-limit` global rate limit flags for assets to agent and backend references.
- Adds global rate limit flag information and link to configuration sections of agent and backend references to the assets reference.

Also propagates these changes to 5.20 docs (so that I don't forget to remove the "added in 5.19.3" caveat)

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2414

### CODE REVIEW

File: content/sensu-go/5.19/reference/agent.md [hunk 1]
```diff
@@ -740,6 +740,8 @@ Flags:
       --annotations stringToString            entity annotations map (default [])
       --api-host string                       address to bind the Sensu client HTTP API to (default "127.0.0.1")
       --api-port int                          port the Sensu client HTTP API listens on (default 3031)
+      --assets-burst-limit int                asset fetch burst limit (default 100) -- Sensu Go 5.19.3
+      --assets-rate-limit float               maximum number of assets fetched per second -- Sensu Go 5.19.3
```
Comment (nikkictl) [inline content/sensu-go/5.19/reference/agent.md]: I think it's fair to note that these flags are introduced in 5.19.3, but maybe just not in the help usage. Would it make sense to make a note about that somewhere else in the reference?

File: content/sensu-go/5.19/reference/agent.md [hunk 2]
```diff
@@ -807,6 +809,32 @@ annotations:
 {{< /highlight >}}
 
 
+| assets-burst-limit   |      |
+--------------|------
+description   | Maximum amount of burst allowed in a rate interval when fetching assets. Added in [Sensu Go 5.19.3][55].
```
Comment (nikkictl) [inline content/sensu-go/5.19/reference/agent.md]: Oh I see, it's here. It's probably best to remove the additional text from the help usage if its documented here!

Comment (hillaryfraley) [reply content/sensu-go/5.19/reference/agent.md]: Yes, I agree. I made the updates...thank you!

**Review by nikkictl — APPROVED** [review]
Excellent work!

**Pull request merged**